ADOPTERS.md

```diff
@@ -215,3 +215,4 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
 | [Inter&Co](https://bancointer.com.br)                                       | [Arnaud Lanna](https://github.com/arnaudlanna), [Adriano Silva](https://github.com/adrianovss), [Bruno Grossi](https://github.com/begrossi)                                                                                                                      | We're using Backstage as our internal Developer Portal to catalog and collect repositories and microservices pieces of information like ownership, deployment time, and documentation.                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | [StatusNeo](https://statusneo.com/)                                          | [Karan Nangru](mailto:nangru@statusneo.com), [@NishkarshRaj](https://github.com/NishkarshRaj), and [Gaurav Sarien](mailto:gaurav.sarien@statusneo.com)                                                                                                                                                                                                                | Harnessing the power of central catalog inventory and self-serving software templates                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| [Contino](https://www.contino.io/) | [Joseph Patterson](mailto:joseph.patterson@contino.io) | Building out a central catalog for our software community |
```

Comment:
Maybe this could go into it's own PR as it's not directly related?

Reply:
sure I'll pull it

Reply:
If you make the PR with just this they will merge it in pretty fast!